tests/unit/test_versioning.py

```diff
@@ -0,0 +1,181 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+
+from typing import Tuple, Any, Type
+
+import pytest
+
+from nucypher.utilities.versioning import Versioned
+
+
+def _check_valid_version_tuple(version: Any, cls: Type):
+    if not isinstance(version, tuple):
+        pytest.fail(f"Old version handlers keys for {cls.__name__} must be a tuple")
+    if not len(version) == Versioned._PARTS:
+        pytest.fail(f"Old version handlers keys for {cls.__name__} must be a {str(Versioned._BRAND_SIZE)}-tuple")
```

Comment:
Is this value supposed to be `_PARTS`?

```suggestion
        pytest.fail(f"Old version handlers keys for {cls.__name__} must be a {str(Versioned._PARTS)}-tuple")
```

Reply:
✔️